internal/cli/atlas/alerts/settings/settings.go

```diff
@@ -63,86 +65,112 @@ type ConfigOpts struct {
 	metricThresholdThreshold        float64
 }
 
-func (opts *ConfigOpts) NewAlertConfiguration(projectID string) *atlas.AlertConfiguration {
-	out := new(atlas.AlertConfiguration)
+func (opts *ConfigOpts) NewAlertConfiguration(projectID string) *admin.AlertConfigViewForNdsGroup {
+	out := new(admin.AlertConfigViewForNdsGroup)
 
-	out.GroupID = projectID
-	out.EventTypeName = strings.ToUpper(opts.event)
+	out.GroupId = &projectID
+	out.EventTypeName = pointer.Get(strings.ToUpper(opts.event))
 	out.Enabled = &opts.enabled
 
 	if opts.matcherFieldName != "" {
-		out.Matchers = []atlas.Matcher{*opts.newMatcher()}
+		out.Matchers = []map[string]interface{}{opts.newMatcher()}
```

Comment:
This looks like downgraded experience from v1 to v2. 
We need jira to fix it by example or openapi fix.